src/api/playlists.js

```diff
@@ -0,0 +1,19 @@
+export default function getPlaylist(){
```

Comment:
This is a collection of playlists, so it should be in plural
```suggestion
export default function getPlaylists(){
```